adam-core/src/main/scala/org/bdgenomics/adam/models/Attribute.scala

```diff
@@ -36,16 +36,20 @@ case class Attribute(tag: String, tagType: TagType.Value, value: Any) {
     val byteSequenceTypes = Array(TagType.NumericByteSequence, TagType.NumericUnsignedByteSequence)
     val intSequenceTypes = Array(TagType.NumericIntSequence, TagType.NumericUnsignedIntSequence)
     val shortSequenceTypes = Array(TagType.NumericShortSequence, TagType.NumericUnsignedShortSequence)
+    val sb = new StringBuilder
+    sb.append(tag)
+    sb.append(":")
```

Comment:
These could be all one line for code style purposes, but I don't mind either way